operator/v1/types_ingress.go

```diff
@@ -1227,6 +1241,17 @@ type IngressControllerLogging struct {
 	//
 	// +optional
 	Access *AccessLogging `json:"access,omitempty"`
+
+	// logLevel describes the desired logging verbosity for Router pods.
+	// Any one of the following values may be specified:
+	// * Normal
```

Comment:
Like in https://github.com/openshift/api/pull/1031/files#diff-46e86efb5c123e387737010ac03828fda0c5edfa402c432480485d928827236cR84-R86 it would be helpful godoc to document what is logged at each of these log levels.

Reply:
I'm not sure I can be as concrete in defining what will be logged at each level, but I will try to give some guidelines/examples